test/helpers/cons.go

```diff
@@ -171,10 +171,9 @@ const (
 	KubectlPolicyNameLabel      = k8sConst.PolicyLabelName
 	KubectlPolicyNameSpaceLabel = k8sConst.PolicyLabelNamespace
 
-	StableImage = "docker.io/cilium/cilium:v1.1.4"
-
-	configMap = "ConfigMap"
-	daemonSet = "DaemonSet"
+	CiliumStableVersion      = "v1.1.4"
```

Comment:
This should be updated to v1.2.1

Reply:
Agreed, but that should be done in a different PR.